plugins/woocommerce-admin/client/products/edit-product-page.tsx

```diff
@@ -0,0 +1,106 @@
+/**
```

Comment:
The logic within this component could be combined and moved into the `add-product-page` component if we wanted to, but to keep things relatively clear I kept them separate for now.